packages/components/addon/components/hds/code-block/index.hbs

```diff
@@ -0,0 +1,30 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class={{this.classNames}} ...attributes>
+  <div class="hds-code-block__header">
+    {{~yield (hash Title=(component "hds/code-block/title"))~}}
+    {{~yield (hash Description=(component "hds/code-block/description"))~}}
+  </div>
+
+  <pre
+    class="hds-code-block__code"
+    {{style maxHeight=@maxHeight}}
+    contentEditable={{not this.isReadOnly}}
```

Comment:
what about calling the `isReadOnly` instead `isContentEditable` which is more semantic, and we can have it `false` by default?

(I know in Cloud UI is called `isReadOnly, but in that case is for the `FieldEditor` so having it readonly by default `false` makes sense, it's an _editor_

Reply:
Renamed in 31583ae447e6c6f3ffef91dc7b2cd8a9c892c0fe. Will update the docs as well.